app/assets/stylesheets/tmix-icon.css

```diff
@@ -23,6 +23,12 @@ i {
   -moz-osx-font-smoothing: grayscale;
 }
 
+.tmix-scaling-down:before {
+  content: "\e96d";
+}
+.tmix-design-finish:before {
+  content: "\e96e";
+}
```

Comment:
Yes. Order is conform by icomoon's generated CSS.

I guess class order is by descending with update.